src/components/TeamMember/TeamMember.tsx

```diff
@@ -39,29 +39,41 @@ const TeamMember = ({ data }: { data: ITeamMember }) => {
     { name: "lattes", href: lattes, icon: "lattes", size: 18 },
   ];
 
+  const handleClick = () => {
+    const firstMedia = socialMedias.find((media) => media.href);
+    if (firstMedia) {
+      window.open(firstMedia.href);
+    }
+  };
+
   return (
     <Wrapper
       htmlFor={name.replace(" ", "_").toLowerCase()}
-      active={(institution || fieldWork)?.toString() || "false"}
+      //active={(institution || fieldWork)?.toString() || "false"}
+      active={"true"}
     >
       <AvatarFrame>
-        <Avatar
-          src={
-            `https:${typeof avatar === "object" ? avatar.fields.file.url : avatar}` ||
-            "avatar.svg"
-          }
-          title={name}
-          alt="Profile picture"
-          width={300}
-          height={300}
-        />
+        <AvatarCircle>
+          <Avatar
+            src={
+              `https:${typeof avatar === "object" ? avatar.fields.file.url : avatar}` ||
+              "avatar.svg"
+            }
+            title={name}
```

Comment:
Would be great if instead of showing the person's name add the `firstMedia.href` here, please